packages/Tag/src/Tag.styles.js

```diff
@@ -120,20 +64,15 @@ export const Ellipsis = styled.div(({ size }) => {
   `;
 });
 
-export const Delete = styled(Button.Close)(({ size }) => {
+export const Delete = styled(TimesCircleIcon)(({ size }) => {
```

Comment:
This `Delete` element should still be a button. You can use the RawButton component and render a TimesCircleIcon inside it 
http://storybooks.highbond-s3.com/paprika/uxd-1786-updateTag/?path=/story/buttons-rawbutton--variations